src/controller/java/templates/CHIPClusters-JNI.zapt

```diff
@@ -276,8 +276,8 @@ class CHIP{{chipCallback.name}}AttributeCallback : public Callback::Callback<{{c
             err = JniReferences::GetInstance().FindMethod(env, javaCallbackRef, "onSuccess", "(Ljava/lang/String;)V", &javaMethod);
             VerifyOrReturn(err == CHIP_NO_ERROR, ChipLogError(Zcl, "Could not find onSuccess method"));
 
-            UtfString valueStr(env, value);
-            env->CallVoidMethod(javaCallbackRef, javaMethod, valueStr.jniValue());
+            jstring valueStr = env->NewString(reinterpret_cast<const jchar *>(value.data()), value.size());
```

Comment:
This doesn't seem right.  `jchar` is a 16-bit type, no?  This is going to run off the end of the buffer, and produce garbage for the part before that....  Seems to me like continuing to use UtfString but with the span would make more sense....